sample_code/ep128/README.md

```diff
@@ -0,0 +1,9 @@
+# [a python plugin system via entrypoints (intermediate)](https://www.youtube.com/watch?v=fY3Y_xPKWNA)
+
+Today I show how to use importlib.metadata + setuptools entrypoints to build a small plugin system!
+
+## Note
+
+[`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html) was added in python 3.8. If you're using an older version, you'll need to install [`importlib_metadata`](https://pypi.org/project/importlib-metadata/), a backport of it.
```

Comment:
Do you have a preferred line length for markdown? Didn't look like it from existing readme's.

Reply:
If you need template/example, you can check the other markdown files in this folder.

Also - using direct hyperlinks is not very good idea, because most links change/break or stop working in the future.
So I guess that simple `importlib.metadata` is enough - this way everyone can easily select the text and search by it if they want more information.

This is how Anthony does it in the videos too -- and usually search engines are better way to access up to date resources, since the search results are almost accessible/reachable.

Reply:
I removed the direct hyperlinks to `importlib.metadata` and `importlib_metadata`.